docs/sdk/paying-for-lit/payment-delegation-db.md

```diff
@@ -167,9 +167,9 @@ Remember that `payerSecretKey` is essentially the private key to your new `payer
 A full implementation of this code for this section can be found [here](https://github.com/LIT-Protocol/developer-guides-code/blob/wyatt/payment-delegation-relayer/payment-delegation-db-relayer/nodejs/src/addUsers.ts).
 
 To add users as `payees` for your `payer` wallet, you're going to need the Relayer API URL for the same Lit network you registered your `payer` on:
-  - For `habanero`, we'll be making requests to:
+  - For `datil-prod`, we'll be making requests to:
       ```
-      https://habanero-relayer.getlit.dev/add-users
+      https://datil-prod-relayer.getlit.dev/add-users
```

Comment:
did you verify that this link is accurate?

Reply:
Yeah  `datil-test` works like this:
`const LIT_RELAYER_URL = 'https://${LIT_NETWORK}-relayer.getlit.dev/register-payer';`

Reply:
ok great

Reply:
I believe the relayer is not supporting it yet though

Reply:
Should have been `datil`, changed, now works